apps/demos/testing/widgets/chat/Customization.test.js

```diff
@@ -0,0 +1,130 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import { Selector } from 'testcafe';
+import { runManualTest } from '../../../utils/visual-tests/matrix-test-helper';
+import { testScreenshot } from '../../../utils/visual-tests/helpers/theme-utils';
+
+fixture('Chat.Customization')
+  .page('http://localhost:8080/apps/demos/Demos/Chat/Customization/jQuery/')
+  .before(async (ctx) => {
+    ctx.initialWindowSize = [900, 600];
+  });
+
+// runManualTest('Chat', 'Customization', ['jQuery'/* , 'React', 'Vue', 'Angular' */], (test) => {
+  test('Sending Message', async (t) => {
+    const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+    await t
+        .typeText('.dx-texteditor-input','testing')
+        .pressKey('enter');
+    await testScreenshot(t, takeScreenshot, 'Chat after sending a message.png');
+
+    await t
+      .expect(compareResults.isValid())
+      .ok(compareResults.errorMessages());
+  });
+
+  test('Test showAvatar', async (t) => {
+    const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+    await testScreenshot(t, takeScreenshot, 'Chat when showAvatar = true.png');
+
+    await t
+        .click('#show-avatar');
+
+    await testScreenshot(t, takeScreenshot, 'Chat when showAvatar = false.png');
+
+    await t
+      .expect(compareResults.isValid())
+      .ok(compareResults.errorMessages());
+  });
+
+  test('Test showUsername', async (t) => {
+    const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+    await testScreenshot(t, takeScreenshot, 'Chat when showUsername = true.png');
```

Comment:
I suggest leaving only the screenshot after changing the option, as the initial values of the fields are tested during demo initialization